configuration/persistence.md

```diff
@@ -232,43 +232,71 @@ Example:
 
 The statement
 
-`Temperature.historicState(now.minusDays(1))`
+`Temperature.persistedState(now.minusDays(1))`
```

Comment:
Currently both arte supported, but `historicState` is deprecated. The reason is that it can also return future states (if the persistence service supports it).